### PR TITLE
Input

### PR DESCRIPTION
**Pull request by ishant005**

### CODE REVIEW

**Comment by arpit01923** [comment]
Add validation to input otherwise looks good

**Review by Bibin-Antony — COMMENTED** [review]
everything seems fine, could move forward.

**Review by Bibin-Antony — APPROVED** [review]

File: input/input.css [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+    padding: 0.1em;
+}
+
+.input > input.failure {
+    border: 2px solid var(--error-color);
+}
+
+.input > div.failure {
+    color: var(--error-color);
+}
```
Comment (laxmikanthkonthum) [inline input/input.css:93]: may be we can use different class names for better readability unless these class names are used in other components

File: input/input.html [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>input</title>
+        <link rel="stylesheet" href="input.css" />
+    </head>
+    <body>
+        <div class="container">
+            <div class="input-group">
+                <label>Username *</label>
+                <input type="text" placeholder="Enter your username" />
```
Comment (laxmikanthkonthum) [inline input/input.html:14]: i think using for attribute for label element is a good practice.

**Review by laxmikanthkonthum — COMMENTED** [review]
Added a couple of comments and rest looks good to me

**Pull request merged**